db/data_migrations/20210817124456_update_validity_end_date.rb

```diff
@@ -0,0 +1,21 @@
+TradeTariffBackend::DataMigrator.migration do
+  name 'Fixes missing validity_end_date for xi measure 3753097'
+
+  up do
+    applicable { false }
+
+    apply do
+      if TradeTariffBackend.xi?
+        end_date = Time.zone.parse('2020-02-26')
+        measure = Measure.where(measure_sid: 3_753_097).last
+
+        return if measure.validity_end_date == end_date
+
+        measure.update(validity_end_date: end_date)
```

Comment:
This assigns the new date if its anything but the expected one? Should it be checking that its blank first?

Reply:
I don't think so. TBH, I'd kind of want to throw this migration away once we're done. I guess the visibility is good.